analysis_runner/cromwell.py

```diff
@@ -386,6 +388,32 @@ def _get_wait_interval(
                 continue
             status = r.json().get('status')
             _remaining_exceptions = max_sequential_exception_count
+
+            # if workflow has concluded print logging to hail batch log only do
+            # once workflow has reached a final state so as not to duplicate
+            if status.lower() in terminal_statuses:
+                logger.info('Cromwell workflow has concluded - fetching log')
+                metadata_url = (
```

Comment:
Keen to get your thoughts, but I'd be keen for using the implementation for getting and displaying metadata from analysis_runner.cli_cromwell (or moving it to Cromwell), then you don't need to know the params of the Cromwell meta call, as you'd probably want to include subworkflows here.

Reply:
This uses that method, indirectly - the `cromwell_cli` method renders the WorkflowMetadataModel from file, but I've set it to parse straight from the received JSON - the `WorkflowMetadataModel.display()` call is the same (line 412 ->)

Reply:
There is the `_check_cromwell_status` method which could be dropped in instead, but the url created doesn't match this one so not sure if they're equivalent